app/services/upscan/UpscanService.scala

```diff
@@ -86,12 +91,15 @@ class UpscanService @Inject()(uploadJourneyRepository: UploadJourneyRepository,
               }
             }
           } else if(System.nanoTime() > timeoutNano) {
-            logger.error("[UpscanService][waitForStatus] - Checking for completed upload status timed out - returning ISE")
-            Future(errorHandler.showInternalServerError)
+            logger.error("[UpscanService][waitForStatus] - Checking for completed upload status timed out - rendering 'taking longer than expected' page")
```

Comment:
can we make this a warning as opposed to an error as the service is handling the issue gracefully so the user can continue the journey so no action is required?
```suggestion
            logger.warn("[UpscanService][waitForStatus] - Checking for completed upload status timed out - rendering 'taking longer than expected' page")
```

Reply:
Yeah makes sense, forgot to change the level